src/CloudFlare/IpRewrite.php

```diff
@@ -2,65 +2,78 @@
 
 namespace CloudFlare;
 
-class IpRewrite {
-    static protected $is_cf = NULL;
-    static protected $original_ip = FALSE;
-    static protected $rewritten_ip = FALSE;
-    
-    static protected $cf_ipv4 = array(
-        "199.27.128.0/21",
-        "173.245.48.0/20",
-        "103.21.244.0/22",
-        "103.22.200.0/22",
-        "103.31.4.0/22",
-        "141.101.64.0/18",
-        "108.162.192.0/18",
-        "190.93.240.0/20",
-        "188.114.96.0/20",
-        "197.234.240.0/22",
-        "198.41.128.0/17",
-        "162.158.0.0/15",
-        "104.16.0.0/12",
-        "172.64.0.0/13"
+require_once 'vendor/autoload.php';
+
+class IpRewrite
+{
+    protected static $is_loaded = false;
+    protected static $original_ip = null;
+    protected static $rewritten_ip = null;
+
+    // Found at https://www.cloudflare.com/ips/
+    protected static $cf_ipv4 = array(
+        '199.27.128.0/21',
+        '173.245.48.0/20',
+        '103.21.244.0/22',
+        '103.22.200.0/22',
+        '103.31.4.0/22',
+        '141.101.64.0/18',
+        '108.162.192.0/18',
+        '190.93.240.0/20',
+        '188.114.96.0/20',
+        '197.234.240.0/22',
+        '198.41.128.0/17',
+        '162.158.0.0/15',
+        '104.16.0.0/12',
+        '172.64.0.0/13',
+        '131.0.72.0/22',
     );
-    
-    static protected $cf_ipv6 = array(
-        "2400:cb00::/32",
-        "2606:4700::/32",
-        "2803:f800::/32",
-        "2405:b500::/32",
-        "2405:8100::/32"
+
+    protected static $cf_ipv6 = array(
+        '2400:cb00::/32',
+        '2606:4700::/32',
+        '2803:f800::/32',
+        '2405:b500::/32',
+        '2405:8100::/32',
     );
-    
-    // Returns boolean
-    public static function isCloudFlare()
+
+    // Helper method for testing, should not be used in production
```

Comment:
I'm not sure how to remove this since it's being used in in [testing](https://github.com/thellimist/cf-ip-rewrite/blob/3950b27100d04c777959e826ba3f7f5b2a71efcd/tests/RewriteTest.php#L9)

If I remove it I must change the 

```
    protected static $is_loaded = false;
    protected static $original_ip = null;
    protected static $rewritten_ip = null;
```

variables to `public` because testing method needs a way to reset these variables.